### PR TITLE
Doctor viewing their patient list

### PR DESCRIPTION
**Pull request by Zain-MKhan**
# General info

**Issue number**:  #10 

**Task description**: 

 - Doctor can now view a list of their patients in one page

# Testing

**Test coverage**:

### /frontend/components/Tabs/MonitorPatients.test.js

**Note to reviewers**:

- Sign in with a doctor account who has patients assigned to them
- Under the 'Monitor Patients' tab found in the doctor-dashboard, you will now find a list containing all the patients assigned to them

### CODE REVIEW

File: frontend/src/components/Dashboards/DoctorDashboard/DoctorDashboard.js [hunk 1]
```diff
@@ -7,21 +7,28 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import styles from "./DoctorDashboard.module.css";
 import "../DashboardCommonStyles.css";
 import NotificationBox from "../../NotificationBox/NotificationBox";
+import axios from "axios";
```
Comment (p-carrier) [inline frontend/src/components/Dashboards/DoctorDashboard/DoctorDashboard.js:10]: Remove unused import axis, useAuth and useState

File: frontend/src/components/Tabs/MonitorPatients.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import { useAuth } from "../Authentication/FirebaseAuth/FirebaseAuth";
+import "bootstrap/dist/css/bootstrap.min.css";
+import styles from "./MonitorPatients.module.css";
+import "./CommonPageStyling.css";
+import axios from "axios";
+import { ListGroup } from "react-bootstrap";
+
+function MonitorPatients(props) {
+
+    // //the following 5 const's are to test the ui without the backend:
```
Comment (p-carrier) [inline frontend/src/components/Tabs/MonitorPatients.js:16]: Do you need the commented text? They should not appear in the master branch

**Comment by AC159** [comment]
Can you pull from the master branch and push this branch again because some changes seem redundant

**Review by p-carrier — CHANGES_REQUESTED** [review]
I test the application in the browser. I can get the patient list of a doctor. That works. Just need to remove things that are not being used.

File: frontend/src/components/Tabs/MonitorPatients.js (hunk 2 above)
Comment (Zain-MKhan) [reply frontend/src/components/Tabs/MonitorPatients.js]: You are right - I removed them!

**Comment by p-carrier** [comment]
Although the test are not passing. It is a bug that only appears on GitHub so I'll accept the merge

**Pull request merged**